src/index.ts

```diff
@@ -182,7 +183,7 @@ export default class PrettyFormatter extends SummaryFormatter {
       (item) => item.id === testStepStarted.testStepId
     )
 
-    if (testStep && testStep.pickleStepId !== '') {
```

Comment:
This is the only change of substance beyond TypeScript compilation stuff, and what is currently broken wiht 7.3.0. Example:

```
TypeError: Cannot read property 'astNodeIds' of undefined
    at PrettyFormatter.onTestStepStarted (/Users/davidgoss/Projects/cucumber-puppeteer-typescript-starter/node_modules/@cucumber/pretty-formatter/src/index.ts:187:36)
    at PrettyFormatter.parseEnvelope (/Users/davidgoss/Projects/cucumber-puppeteer-typescript-starter/node_modules/@cucumber/pretty-formatter/src/index.ts:137:12)
    at EventEmitter.emit (events.js:326:22)
```

With the change in `messages` from using protobuf to JSON schema, some fields that used to be defaulted to an empty string (per protobuf convention) are now undefined if not set. Making this a simple truthiness check solves the issue and is still backwards-compatible with the older version.